crypto/modes/ocb128.c

```diff
@@ -74,7 +74,8 @@ static void ocb_double(OCB_BLOCK *in, OCB_BLOCK *out)
      */
     mask = in->c[0] & 0x80;
     mask >>= 7;
-    mask *= 135;
+    mask *= -1;
+    mask &= 135;
```

Comment:
If one aims for multiplication removal, then I'd rather suggest `mask = (0 - mask) & 135`. It would be more readable, for following reason. There are implicit type conversions that take place here, and multiplication by -1 triggers more elaborate mental path to confirm that course of operations is free from undefined behaviour as defined by language standard. I would also suggest to replace 135 with hexadecimal for better readability.

Reply:
Done.